src/definitions.ts

```diff
@@ -97,13 +97,16 @@ export interface Config {
 
   marketplace?: {
     // Offer service path
-    offer?: string
+    offer: string
```

Comment:
But why. It's mandatory to have it?

Reply:
Done

Reply:
Well if you would like to use it somewhere as partial typing then it is a problem, because the `Partial<>` helper makes things optional only on first level, hence why I keep the whole `Config` with `?`